src/sqlancer/common/DBMSCommon.java

```diff
@@ -39,4 +40,26 @@ public static int getMaxIndexInDoubleArray(double... doubleArray) {
         }
         return maxIndex;
     }
+
+    public static int areQueryPlanSequencesSimilar(List<String> list1, List<String> list2) {
```

Comment:
What I meant is that we have a separate `areQueryPlanSequencesSimilar` method that returns a boolean value, and the previous `private` method that is called by it.

Reply:
Sorry for the misunderstanding. Fixed now.